lib/Twig/Extension/Core.php

```diff
@@ -852,6 +852,30 @@ function twig_in_filter($value, $compare)
     return false;
 }
 
+/** 
+ * Returns a trimmed string.
+ * 
+ * @param string $string
+ * @param string $character_mask
+ * @param string $mode
+ * @return string
+ */
+function twig_trim_filter($string, $character_mask = null, $type = 'both')
+{
+    if (null === $character_mask) {
+        $character_mask = " \t\n\r\0\x0B";
+    }
+    switch ($type) {
+        case 'left':
+            return ltrim($string, $character_mask);
+        case 'right':
+            return rtrim($string, $character_mask);
+        case 'both':
+        default:
```

Comment:
Wouldn't it be better to throw an exception here instead of fallbacking to `both`, which is already the default value of `$type` parameter anyway?